dolma/src/tokens/types/emoji.ts

```diff
@@ -0,0 +1,10 @@
+import { Token } from "../../util/types/tokenTypes";
+import emojiRegex from "emoji-regex"
+
+export default {
+	name: "emote",
```

Comment:
the name for this token is `emoji` not `emote`

Reply:
```suggestion
	name: "emoji",
```